spec/models/language/tag_spec.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Language::Tag do
+  let(:iana_en_fixture) { file_fixture('files/iana_language/english-subtag-registry') }
+  let(:iana_i_fixture) { file_fixture('files/iana_language/grandfathered-subtag-registry') }
+  let(:iana_qu_fixture) { file_fixture('files/iana_language/quechua-subtag-registry') }
+  let(:iana_zh_fixture) { file_fixture('files/iana_language/chinese-subtag-registry') }
+  let(:use_preferred) { false }
+  let(:lang_tag) { described_class.for(tag_value, use_preferred) }
+  let(:preferred_value) { lang_tag.preferred_value }
+  before do
+    iana_fixtures.each do |iana_fixture|
+      Hyacinth::Language::SubtagLoader.new(iana_fixture).load
+    end
+  end
+  context 'tag has only a language' do
+    let(:iana_fixtures) { [iana_en_fixture] }
+    let(:tag_value) { 'en' }
+    it "associates a suppressed script without including it in the tag" do
+      expect(lang_tag.tag).to eql tag_value
+      expect(lang_tag.suppressed_script&.subtag).to eql 'Latn'
+    end
+  end
+  context 'tag has a region' do
+    let(:iana_fixtures) { [iana_en_fixture] }
+    let(:tag_value) { 'en-US' }
+    it "associates a suppressed script without including it in the tag" do
+      expect(lang_tag.tag).to eql tag_value
+      expect(lang_tag.suppressed_script&.subtag).to eql 'Latn'
```

Comment:
Why use the safe navigation operator after `suppressed_script` here if you're expecting a known test value?

Reply:
Same question for three other tests in this file that use `&.`.